src/bundle/Resources/public/scss/_card.scss

```diff
@@ -1,6 +1,6 @@
 .ibexa-card {
     border: none;
-    background-color: $ibexa-color-white;
+    background-color: transparent;
```

Comment:
This should be fine given there is `.ibexa-card--light` with `background: $ibexa-color-white;`.